src/migration.ts

```diff
@@ -82,7 +135,14 @@ async function getLastSuffix(
   }
 }
 
-export function getTimestamp(logger: Logger, filename: string): number {
+/**
+ * ! this function does not only resolve a timestamp numeric value. When prefix is numeric but neither 13 nor 17 digits, it returns the numeric prefix.
+ * ? should the function body be adjusted to do only what can be expected from it's name
```

Comment:
**todo:** @Shinigami92 needs to checkout the code/code-path and evaluate  
_resolve open question_

Reply:
since this question is *only* about naming, it shouldn't be a showstopper